calico_node/startup/startup.go

```diff
@@ -341,6 +341,28 @@ func validateIP(ipn *net.IPNet) {
 	warning("Unable to confirm IPv%d address %s is assigned to this host", ip.Version(), ip)
 }
 
+// evaluateENVBool evaluates a passed environment variable
+// Returns True if the envVar is defined and set to true.
+// Returns False if the envVar is defined and set to false.
+// Returns def in the envVar is not defined.
+// An log entry will always be written
+func evaluateENVBool(envVar string, defaultValue bool) bool {
+	envValue, isEmpty := os.LookupEnv(envVar)
+
+	log.Info("%s is %s through environment variable", envVar, envValue)
```

Comment:
I think you should log the value that this function is returning, that way if the user is setting the value incorrectly it will be obvious if the variable is not being interpreted as they expect.
I think it would be nice to also include "(defaulted)" in the log message if the default value is being used.

You need to use `log.Infof` to make a formatted message, otherwise you get `%s is %s through environment variableCALICO_IPV4POOL_NAT_OUTGOING` as the output